dockerfiles/spark/Dockerfile

```diff
@@ -1,13 +1,13 @@
-FROM alpine:3.20
+FROM alpine:3.22
 
-ENV SPARK_VERSION=3.5.1 \
+ENV SPARK_VERSION=3.5.6 \
```

Comment:
even upstream only supports 3.5.1
so I am not going to merge above, it would need more extensive testing, checking what changed if there aren't any incompats

@nopzdk can you take above change away?
I will be able to merge the rest